File: fixtures/dotnet-pilot/src/Api/OrdersController.cs
namespace Pilot.Api;

using Pilot.Orders;

public class OrdersController
{
    private readonly OrderService _orders;

    public OrdersController(OrderService orders)
    {
        _orders = orders;
    }

    public void create(string orderId)
    {
        _orders.PlaceOrder(orderId);
    }
}

File: fixtures/php-pilot/src/Api/OrdersController.php
<?php

require_once __DIR__ . '/../Orders/OrderService.php';

class OrdersController
{
    private OrderService $orders;

    public function __construct(OrderService $orders)
    {
        $this->orders = $orders;
    }

    public function create($orderId): void
    {
        $this->orders->PlaceOrder($orderId);
    }
}
